README.md

```diff
@@ -1,36 +1,108 @@
 # Learn GRASS GIS
 
-Welcome to the Learn GRASS GIS repository that hosts a Quarto-powered website
-with a collection of tutorials and learning materials for [GRASS
-GIS](https://grass.osgeo.org), an open-source platform for geospatial analysis
-and modeling. The tutorials cover a variety of topics and is designed to help
-users at different skill levels.
+## 📌 What is GRASS GIS?
```

Comment:
Keep the current intro, no need to have detailed explanation what is GRASS, this is not GRASS repo.